satellite/console/usercredits_test.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 func TestUserCredits(t *testing.T) {
+	t.Skip("Skip until usercredits.Create method is cockroach compatible")
```

Comment:
might be helpful to link the issue about the cockroachdb error, the `:::` thingy